contracts/shared/helpers/TokenHelper.ts

```diff
@@ -56,7 +57,7 @@ export default class TokenHelper {
   async walletTokenSetup(): Promise<BlsWalletWrapper[]> {
     const wallets = await this.fx.createBLSWallets();
 
-    this.testToken = await TokenHelper.deployTestToken();
+    this.testToken = (await TokenHelper.deployTestToken()) as IERC20;
```

Comment:
You could modify [TokenHelper.deployTestToken](https://github.com/jzaki/bls-wallet/blob/main/contracts/shared/helpers/TokenHelper.ts#L22-L29) to use the Typechain factory and contract type instead of doing the type cast here.